050-AIS-APIManagementwithFunctions/Student/Challenge-04.md

```diff
@@ -0,0 +1,73 @@
+# Challenge 04 - Securing Backends APIs
+
+
+[<Previous Challenge](./Challenge-03.md) - **[Home](../README.md)**
+
+## Pre-requisites
+
+- You should have completed Challenge 03
+
+## Introduction
+You would like to be able to secure your backend APIs in one of the two ways:
+- Secure Hello API in a private network
+- Test end-to-end authorization to Hello API via OAuth.  
+
+## Description
+
+### Scenario 01: Configure secured backend APIs in a private network
+- Create a new Function App in Elastic Premium plan which will be imported to APIM as Hello Internal API.  
+- The existing API - Hello API - will now become the public/external API.  The new path should configured in APIM as: `https://apim-{{unique_id}}.azure-api.net/external/hello`
```

Comment:
Grammar - The new path should be configured

Reply:
Done